src/errors/parser.ts

```diff
@@ -18,7 +18,7 @@ export const getProcessingErrorCodeByStatus = (status: number): string => {
         306: ResponseErrorCode.CardRejected,
         307: ResponseErrorCode.CardInvalid,
         308: ResponseErrorCode.ChargeInvalidData,
-        309: ResponseErrorCode.ExpirationDateError,
```

Comment:
remove `ExpirationDateError` from `ResponseErrorCode` enum if it's not used somewhere else?

Reply:
This is an error from the API. This should not be removed.